reader/views.py

```diff
@@ -224,6 +224,7 @@ def base_props(request):
             "layoutTanakh":  request.COOKIES.get("layoutTanakh", "segmented"),
             "aliyotTorah":   request.COOKIES.get("aliyotTorah", "aliyotOff"),
             "vowels":        request.COOKIES.get("vowels", "all"),
+            "punctuationTalmud": request.COOKIES.get("punctuationTalmud", "punctuationOn"),
```

Comment:
Minor style point: for readability the values of each of these keys were aligned with spaces. By adding a longer key, the other lines should all have spaces added so that each "request" lines of vertically.